modules/ROOT/pages/tgw-about.adoc

```diff
@@ -0,0 +1,49 @@
+= Transit Gateway Attachments
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+image:logo-cloud-active.png[link="/runtime-manager/deployment-strategies#cloudhub", title="CloudHub"]
+image:logo-hybrid-disabled.png[link="/runtime-manager/deployment-strategies#hybrid", title="Hybrid"]
+image:logo-server-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-platform-private-cloud-edition", title="Anypoint Platform PCE"]
+image:logo-rtf-disabled.png[link="/runtime-manager/deployment-strategies#anypoint-runtime-fabric", title="Runtime Fabric"]
+
+A transit gateway is a network hub that connects your on-premises networks to your VPCs.
+The transit gateway simplifies your network connectivity by functioning
+as a cloud router, managing your VPC connections and requiring only one connection from CloudHub.
+
+Use transit gateway attachments to create a secure connection between your Virtual Private Cloud (VPC) and your private corporate networks.
+You can add multiple transit gateway attachments if required. 
+
+[NOTE]
+The number of transit gateways you can add depends on the transit gateway entitlements available to your account.
```

Comment:
The number of transit gateways or attachments?
I think we might need to make a distinction or limit use to the best of the two terms, if they mean the same thing.

Reply:
My understanding is that the customer's entitlement is for a certain number of TGWs, but we don't impose a limit on the number of VPCs they can attach to each TGW. Vinay can correct me if I got this wrong.

I read elsewhere in this doc (I think) that TGWs are technically limited to 10 attachments each. I assumed that a constraint imposed by AWS, not us.

Reply:
Clarified.
The 10 attachments per TGW is an AWS constraint.